demos/python_demos/handwritten_text_recognition_demo/models.lst

```diff
@@ -1,2 +1,3 @@
 # This file can be used with the --list option of the model downloader.
 handwritten-japanese-recognition-0001
+handwritten-simplified-chinese-recognition-0001
```

Comment:
```suggestion
# handwritten-simplified-chinese-recognition-0001  # TODO: reenable this after the model is added
```

Reply:
When will the model will be added? Should I delete handwritten-simplified-chinese-recognition-0001 for now?

Reply:
it is special comment for our automatic checks before release happend. Model will be uploaded with the next OMZ models package, so after that we get a signal to uncomment it. I suggest you to use this feature.